src/pages/extension-manager/extension-developed-by-adobe/configurable-asset-picker/index.md

```diff
@@ -131,16 +131,9 @@ This is sample asset picker configuration file that allows filtering assets by i
 
 ```
 {
-    "aemTierType": [
-        "delivery",
-        "author"
-    ],
```

Comment:
The default configuration is the correct and should also not be modified by most users.